coalib/bearlib/abstractions/Linter.py

```diff
@@ -447,10 +447,13 @@ def run(self, filename, file, **kwargs):
                     FunctionMetadata.filter_parameters(
                         self._get_create_arguments_metadata(), kwargs))
 
-                arguments = (self.get_executable(),) + tuple(
-                    self.create_arguments(
-                        filename, file, config_file,
-                        **create_arguments_kwargs))
+                args = self.create_arguments(filename, file, config_file,
+                                             **create_arguments_kwargs)
+
+                if args is None:  # pragma: no cover
+                    self.warn("The tuple of arguments cannot be None.")
```

Comment:
1. This should be tested!
2. DOn't you want to do a return here or something instead of just running into 456 throwing an exception? ALso this should be an err IMO.

CC @Makman2